canonical-kubernetes/addons/jfrog/metadata.yaml

```diff
@@ -0,0 +1,4 @@
+friendly-name: JFrog Artifactory
+description: |
+  JFrog artifactory provides binary repository 
```

Comment:
JFrog Artifactory provides a binary repository?

Reply:
Yes, we'll update with the correct marketing tagline :). Via this add-on, Artifactory acts as a Kubernetes registry and supports helm, docker and other package types.

Reply:
Please use, "Artifactory is your Kubernetes Registry - the only enterprise-ready universal repository manager supporting secure, clustered, high availability Docker registries, Helm, NPM, and many other repositories."